tests/test_cases/test_cocotb/Makefile

```diff
@@ -29,8 +29,23 @@
 
 include ../../designs/sample_module/Makefile
 
-MODULE := test_cocotb,test_deprecated,test_doctests
+MODULE := "\
+	test_cocotb,\
+	test_deprecated,\
+	test_doctests,\
+	test_synchronization_primitives,\
+	test_concurrency_primitives,\
+	test_tests,\
+	test_generator_coroutines,\
+	test_timing_triggers,\
+	test_scheduler,\
+	test_clock,\
+	test_edge_triggers,\
+	test_async_coroutines,\
+	test_handle,\
+	test_logging,\
+	"
```

Comment:
Maybe as a follow-up we should think of enabling space-separated test lists being passed to MODULE to make all of this less ugly.

Reply:
Would it really make much difference?

I think using a `define` block would eliminate the need for the backslashes

Reply:
I tried that, but how the makefile uses the variable means it isn't interpreted correctly and tries to run the string as a command.

Reply:
Using `export MODULE` should fix that. Quotes at the use sie might too